substrate/client/cli/src/params/node_key_params.rs

```diff
@@ -79,22 +80,47 @@ pub struct NodeKeyParams {
 	/// the chosen type.
 	#[arg(long, value_name = "FILE")]
 	pub node_key_file: Option<PathBuf>,
+
+	/// Forces key generation if node-key-file file does not exist.
+	///
+	/// This is an unsafe feature for producation networks because the previous
+	/// node address would still live in the DHT for 36h and the node would suffer
+	/// from bad connectivity to other nodes untill the old record expires.
```

Comment:
```suggestion
```

Would not mention the DHT, because this is not being true for collators for example. Generally the doc could be improved a little bit.

Reply:
>Generally the doc could be improved a little bit

Improved it a little bit, let me know if it is good now.